migration/migrator/data/submitty_db.sql

```diff
@@ -441,6 +441,7 @@ CREATE TABLE public.courses (
     status smallint DEFAULT 1 NOT NULL,
     group_name character varying(255) NOT NULL,
     owner_name character varying(255) NOT NULL,
+    self_registration_allowed smallint DEFAULT 0,
```

Comment:
For later reviewers, this is an integer because eventually there will be multiple 'types' of self registration allowed, the current way, 'request to join', and maybe others.

Reply:
Thanks for the clarification!  The name `self_registration_allowed` feels like a binary option.  Would it be better to rename it to `registration_status` or similar?

Reply:
I agree that it feels like a boolean yes or no, we can discuss another name in the meeting.